flaskbb/forum/forms.py

```diff
@@ -58,12 +58,6 @@ class NewTopicForm(ReplyForm):
     title = StringField(_("Topic title"), validators=[
         DataRequired(message=_("Please choose a title for your topic."))])
 
-    content = TextAreaField(_("Content"), validators=[
-        DataRequired(message=_("You cannot post a reply without content."))])
-
-    track_topic = BooleanField(_("Track this topic"), default=False,
-                               validators=[Optional()])
-
     submit = SubmitField(_("Post Topic"))
```

Comment:
This should probably vary based if it's edit/new. Might be easier to have a base TopicForm and then have a NewTopicForm and EditTopicForm. If we do this, I'd recommend seeing if making `submit` an abstract property is possible to avoid instantiation of the parent class.